internal/lox_files_test.go

```
@@ -0,0 +1,126 @@
+				line = pattern.ReplaceAllString(line, replacement)
+			}
+
+			if len(line) > maxLineLength {
+				t.Errorf("Line length exceeds %d in file: %s, line number: %d, line length: %d",
+					maxLineLength, file, i+1, len(line))
+			}
+		}
+	}
+}
+func TestNoTrailingSpaces(t *testing.T) {
+	os.Chdir("../test_programs")
```

golangci-lint (1.64.8)

Error return value of `os.Chdir` is not checked

(errcheck)